source/includes/k8s/steps-configure-ad-ldap-external-identity-management.rst

```diff
@@ -4,24 +4,9 @@ Deploy MinIO Tenant with Active Directory / LDAP Identity Management
 1) Access the Operator Console
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Use the :mc-cmd:`kubectl minio proxy` command to temporarily forward traffic between the local host machine and the MinIO Operator Console:
+Use something to temporarily forward traffic between the local host machine and the MinIO Operator Console:
```

Comment:
something like .... nodePorts?
Seems like you're still determining what that is. Need to update it before it merges.